### ISSUE TITLE
Issue running SAIGE for single SNP or region

### ISSUE THREAD

**Issue by ealopera** [weizhouUMICH/SAIGE#225]
Hello,
I am trying to use SAIGE for a single SNP or region from a .bgen imputed file. If I don't use any filter everything runs smoothly and I get proper summary statistics for the whole file (which is a whole chromosome in this case). but if I use any of the filters (--idstoIncludeFile , --rangestoIncludeFile) then I get this error message:

```
1 markers will be analyzed
R: ../src/View.cpp:232: const std::vector<unsigned char>& genfile::bgen::View::read_and_uncompress_genotype_data_block(): Assertion `m_state == e_ReadyForProbs' failed.
Aborted
```
or this error message:

```
70 markers will be analyzed
R: ../src/View.cpp:232: const std::vector<unsigned char>& genfile::bgen::View::read_and_uncompress_genotype_data_block(): Assertion `m_state == e_ReadyForProbs' failed.
Aborted
```

this is how my file with ids or ranges looks to include looks:

```
rs8176645
```

```
9 136149098 136153000
```

and this is how my command looks

```
  singularity exec instance://saige4  step2_SPAtests.R \
      --bgenFile=/tmp/imputed/bgen/${chr}.UGLI.imputed.qc.bgen \
      --bgenFileIndex=/tmp/imputed/bgen/${chr}.UGLI.imputed.qc.bgen.bgi \
      --sampleFile=/tmp/imputed/bgen/${chr}.UGLI.imputed.qc.bgen.samples \
      --minMAF=0.01 \
      --minInfo=0.4 \
      --rangestoIncludeFile=/tmp/region \
      --GMMATmodelFile=/tmp/output/${outdir}/${mb}_gwas/model.rda \
      --varianceRatioFile=/tmp/output/${outdir}/${mb}_gwas/model.varianceRatio.txt \
      --SAIGEOutputFile=/tmp/output/${outdir}/${mb}_gwas/p-values.chr.${chr} \
      --numLinesOutput=2 \
      --IsOutputAFinCaseCtrl=FALSE 
```

As I see is reading correctly the filter files, but the filter suddenly creates this error. Any clues of what might be happening?
thank you

**Comment by weizhouUMICH** [comment]
This error is probably due to the sqlite library. Did you see this error for every query job? 
We've seen this error once before when many query jobs were started at the same time. When we pauses 1 min to start the next job, the issue was solved. Could you please try this?

**Comment by weizhouUMICH** [comment]
We have just released a new version 1.0.0. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
